G/GnuTLS/build_tarballs.jl

```diff
@@ -1,22 +1,22 @@
 using BinaryBuilder
 
 name = "GnuTLS"
-version = v"3.6.13"
+version = v"3.6.15"
```

Comment:
There's also been a 3.7.0 release in the meantime, but I won't touch that for now, as it has "new features and behavior changes", so that seems to risky for now.